src/inausoft.netCLI.Tests/RootCommandHandlerTests.cs

```diff
@@ -11,30 +11,27 @@ public void RootCommandHandler_RunsProperCommandHandler_ForMultipleOptions()
         {
             //Arrange
             var stringOptionValue = "sampleString";
-            var pathOptionValue = @"C:\Program%Files";
             var intOptionValue = 102;
 
             var args = new string[] 
                             { 
                                 "command1",
                                 "--boolOption",
                                 "--stringOption", stringOptionValue,
-                                "--pathOption", pathOptionValue,
                                 "--intOption", intOptionValue.ToString()
                             };
 
             var mockCommandHandler = new MockCommand1Handler();
-            var rootCommandHanlder = new RootCommandHandler(new ICommandHandler[] { mockCommandHandler });
+            var config = new CLIConfiguration().Map<Command1, MockCommand1Handler>();
 
             //Act
-            var result = rootCommandHanlder.Run(args);
+            var result = netCLI.RunCLI(config, args, mockCommandHandler);
```

Comment:
I think that create the `RunCLI(config, args, commandHandler)` method solely for testing isn't good idea. I would reather test the `IServiceCollection` extension method as described here: https://stackoverflow.com/questions/51180523/unit-testing-iservicecollection-registration

Reply:
There are tests that uses DI already. Also that method was created to allow using .netCLI without DI - however as using netCLI with DI wasn't my focus it could be improved. Take a look at README.md file:
```cs
ICommandHandler[] handlers = new ICommandHandler[] { 
    new MoveCommandHandler() 
};

var config = new CliConfiguration().Map<MoveCommand, MoveCommandHandler>();

return netCLI.RunCLI(config, args, handlers);

```

Reply:
Agreed, but now it means that solution lacking tests on the `RunCLI` method for DI.

Reply:
Not ture. There is a test for it in netCLI.Tests line 41 - have a look.